src/main/groovy/org/fundacionjala/gradle/plugins/enforce/tasks/salesforce/filter/Filter.groovy

```diff
@@ -0,0 +1,74 @@
+package org.fundacionjala.gradle.plugins.enforce.tasks.salesforce.filter
+
+import org.fundacionjala.gradle.plugins.enforce.utils.Constants
+import org.fundacionjala.gradle.plugins.enforce.utils.Util
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.MetadataComponents
+import org.gradle.api.Project
+import org.gradle.api.file.FileTree
+
+class Filter {
+    private Project project
+    private String projectPath
+
+    Filter(Project project, String projectPath) {
+        this.project = project
+        this.projectPath = projectPath
+    }
+
+    /**
+     * Gets a criteria to folders files and wildcards
+     * @param criterion is an String with criteria
+     * @return an ArrayList of criteria
+     */
+    public static ArrayList<String> getCriteria(String criterion) {
+        ArrayList<String> criteria = new ArrayList<String>()
+        criterion.split(Constants.COMMA).each { String critery ->
+            if (!critery.contains('.')) {
```

Comment:
improve this condition because a folder name can has "." for example en documents/